app/admin/dashboard.rb

```diff
@@ -16,10 +14,20 @@
       end
 
       column do
-        panel "Info" do
-          para "Welcome to ActiveAdmin."
+        panel 'Info' do
+          para 'Welcome to ActiveAdmin.'
         end
       end
     end
+
+    section 'This month' do
+      table_for Event.where('datetime > ? and datetime < ?',
```

Comment:
events.where(datetime: DateTime.now.beginning_of_day..
                 DateTime.now.end_of_day)
note: in this case we use including range